device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/DeviceClientTest.java

```diff
@@ -1995,7 +2041,7 @@ private void deviceClientInstanceExpectation(final IotHubClientProtocol protocol
                 break;
         }
 
-        new NonStrictExpectations()
+        new Expectations()
         {
```

Comment:
NonStrictExpectations is deprecated and guidance says to use Expectations instead.